src/main/java/Reporter.java

```diff
@@ -11,7 +11,39 @@
  */
 public class Reporter {
     public static final String REPORT_SUFFIX = "_Report.txt";
+    public static final int REPORT_ALL          = 0xFFFFFFFF;
+    public static final int NUM_PROJECTS        = 0b1;
```

Comment:
I think I would prefer if we made all bitmasks 8 bits long so that it was easier to compare them with one another visually. Not really important though.